erpnext/hr/doctype/employee/employee.json

```diff
@@ -410,6 +410,7 @@
    "options": "Branch"
   },
   {
+   "fetch_from": "grade.default_leave_policy",
```

Comment:
`Fetch If Empty` should also be checked otherwise any other policy selected by user will get overridden by the default policy

Reply:
oh okay cool